src/_tutorials/web/get-started.md

```diff
@@ -3,7 +3,11 @@ title: "Get started: web apps"
 description: A guide to get you quickly writing web apps in Dart.
 ---
 
-Follow these steps to start using Dart to develop web apps.
+Follow these steps to start using Dart to develop **web-only** apps.
+If you want to write a **multi-platform** app, then try
+[Flutter.]({{site.flutter}})
```

Comment:
I think we should link to http://flutter.dev/web here, given this is a web-specific tutorial

Reply:
Done. I've expanded the link to "try Flutter" so people won't necessarily expect to land on the Flutter homepage.